websites/L/LOAP/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "1214632755158319206",
+		"name": "arkhanths"
+	},
+	"service": "LOAP",
+	"description": {
+		"en": "Unofficial Discord Rich Presence for League of Angels: Pact. Works with R2Games and Game Hollywood."
+	},
```

Comment:
This should be a description of what LOAP is, not the Presence.
If possible, get an official description of a website